frontend/device/kobo/device.lua

```diff
@@ -1235,12 +1235,14 @@ function Kobo:toggleChargingLED(toggle)
     --       In fact, Nickel itself doesn't provide this feature on said older devices
     --       (when it does, it's an option in the Energy saving settings),
     --       which is why we also limit ourselves to "true" on devices where this was tested.
+    --       FWIW, on older devices, the knob is at "/sys/devices/platform/pmic_light.1/lit".
     -- c.f., drivers/misc/ntx_misc_light.c
     local f = io.open("/sys/devices/platform/ntx_led/lit", "we")
     if not f then
         logger.err("cannot open /sys/devices/platform/ntx_led/lit for writing!")
         return false
     end
+    C.setlinebuf(f)
```

Comment:
Why is setlinebuf only here?
I would set it globally (or disable it at the end of this method ).

Reply:
It's per (open) FILE handle ;).

Reply:
Oh, of course, you are right.